src/main/java/io/getunleash/metric/ClientRegistration.java

```diff
@@ -23,6 +28,10 @@ public class ClientRegistration implements UnleashEvent {
         this.started = started;
         this.strategies = strategies;
         this.interval = config.getSendMetricsInterval();
+        this.specVersion = config.getClientSpecificationVersion();
+        this.platformName = System.getProperty("java.vm.name", "JRE");
+        this.platformVersion = System.getProperty("java.vm.version", "1.8");
```

Comment:
What should our defaults be if we can't resolve these. Since we know we're requiring Java 1.8, I thought saying 1.8 was safe, but for the platform name (not really sure)

Reply:
The Unleash side will handle nulls and transform them into something safe so I think null is a better choice

Reply:
Handled :)